File: docs/source/examples/Relationships/Inverse/Inverse.swift
class User: Object {
    @objc dynamic var _id: ObjectId = ""
    @objc dynamic var _partition: String = ""
    @objc dynamic var name: String = ""
    let tasks = List<Task>()
    
    override static func primaryKey() -> String? {
        return "_id"
    }
}

class Task: Object {
    @objc dynamic var _id: ObjectId = ""
    @objc dynamic var _partition: String = ""
    @objc dynamic var text: String = ""
    let assignee = LinkingObjects(fromType: User.self, property: "tasks")
    
    override static func primaryKey() -> String? {
        return "_id"
    }
}

File: docs/source/examples/Relationships/Inverse/Inverse.js
const User = {
  name: "User",
  primaryKey: "_id",
  properties: {
    _id: "objectId",
    _partition: "string",
    name: "string",
    tasks: "Task[]"
  }
};

const Task = {
  name: "Task",
  primaryKey: "_id",
  properties: {
    _id: "objectId",
    _partition: "string",
    text: "string",
    assignee: {
      type: 'linkingObjects',
      objectType: 'User',
      property: 'tasks'
    }
  }
};
